docs/source/intro.rst

```diff
@@ -31,26 +31,25 @@ and unnoticed precision lost for identifier is what we want to avoid.
 Authentication
 ''''''''''''''
 
-Authentication to the Google ``BigQuery`` service is via ``OAuth 2.0``.
-Is possible to authenticate with either user account credentials or service account credentials.
+Authentication to the Google ``BigQuery`` service via ``OAuth 2.0``
+is possible to authenticate with either user or service account credentials.
```

Comment:
Authentication to the Google ``BigQuery`` service via ``OAuth 2.0`` is possible ~~to authenticate~~ with either user or service account credentials.